builder/parser/line_parsers.go

```diff
@@ -10,13 +10,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"strconv"
 	"strings"
 	"unicode"
 )
 
 var (
-	errDockerfileJSONNesting = errors.New("You may not nest arrays in Dockerfile statements.")
+	errDockerfileNotStringArray = errors.New("When using JSON array syntax, arrays must be comprised of strings only.")
```

Comment:
:o) Hm, okay.. might be just me then? LOL according to that "bye" is 330% more readable than "hello" :smile: